plugins/docker/docker.plugin.zsh

```diff
@@ -0,0 +1,42 @@
+alias di='docker info'
```

Comment:
Is this really necessary? How often do you use it to warrant an alias?

Reply:
Removed it.